src/aibs_informatics_aws_lambda/handlers/demand/model.py

```diff
@@ -64,6 +65,10 @@ class DataSyncConfiguration(SchemaModel):
 @dataclass
 class ContextManagerConfiguration(SchemaModel):
```

Comment:
Can you add a docstring to describe this class in more detail? The latest set of changes add another set of fields for the new "deletion" feature, so it would be good to have a thorough description of all the "contexts" this class "manages".